fools/py/main.py

```
@@ -0,0 +1,217 @@
+# -----------------------------
+
+if __name__ == "__main__":
+    print("=== Utilities ===")
+    print("Factorial(5):", factorial(5))
+    print("Fibonacci(10):", fibonacci(10))
+    print("Is 29 prime?", is_prime(29))
+    print("Random string:", generate_random_string(12))
+
+    print("\n=== Data Structures ===")
+    s = Stack()
+    s.push(1); s.push(2); s.push(3)
```

Ruff (0.14.13)

Multiple statements on one line (semicolon)

(E702)